eng/targets/ReferenceAssembly.targets

```
@@ -76,10 +77,15 @@
     </ItemGroup>
 
     <PropertyGroup>
+      <ManualReferenceAssemblyContent Condition="'$(ContainsManualReferenceAssemblyContent)' == 'true'">
+      <![CDATA[
+    <Compile Include="$(_ManualRefSourceFileName)" Condition="Exists('$(_ManualRefSourceFileName)')" />]]>
```

```suggestion
    <Compile Include="$(_ManualRefSourceFileName)" />]]>
```
The condition in the generated project isn't necessary. Check for existence only when setting `$(ManualReferenceAssemblyContent)`.